case-lib/lib.sh

```diff
@@ -38,10 +38,9 @@ if [ ! "$SOFCARD" ]; then
     SOFCARD=$(grep '\]: sof-[a-z]' /proc/asound/cards|awk '{print $1;}')
 fi
 
-func_lib_setup_kernel_last_line()
+func_lib_setup_kernel_last_timestamp()
```

Comment:
I think you could have kept this function name for now, it would have made this (risky) refactoring so much smaller. Too late.

Reply:
it's just a rename, not a refactoring.